compiler/optimizer/OMROptimizationManager.cpp

```diff
@@ -49,7 +49,7 @@ OMR::OptimizationManager::OptimizationManager(TR::Optimizer *o, OptimizationFact
         _numPassesCompleted(0),
         _optData(NULL),
         _optPolicy(NULL),
-        _enabled(false),
+        _enabled(!o->comp()->isDisabled(optNum)),
```

Comment:
I'm not sure I agree that _enabled(!o->comp()->isDisabled()) is necessarily more correct than _disabled(o->comp()->isDisabled()), especially given the common case is that every optimization in the strategy is enabled unless disabled.

Q: If an opt has been disabled, do we still need to allocate an entire OptimizationManager for it so that it can tell anyone who asks (optimizer?) that it's not enabled?